src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -3736,8 +3736,8 @@ Copyright (C) Microsoft Corporation. All rights reserved.
           Condition="'@(_SatelliteAssemblyResourceInputs)' != '' and '$(GenerateSatelliteAssembliesForCore)' != 'true'">
 
     <PropertyGroup>
-      <_SdkToolsPathMaybeWithx64Architecture>$(TargetFrameworkSDKToolsDirectory)</_SdkToolsPathMaybeWithx64Architecture>
-      <_SdkToolsPathMaybeWithx64Architecture Condition="'$(PlatformTarget)' == 'x64'">$(TargetFrameworkSDKToolsDirectory)$(PlatformTarget)\</_SdkToolsPathMaybeWithx64Architecture>
+      <ALExeToolPath>$(TargetFrameworkSDKToolsDirectory)</ALExeToolPath>
+      <ALExeToolPath Condition="'$(PlatformTarget)' == 'x64'">$(TargetFrameworkSDKToolsDirectory)$(PlatformTarget)\</ALExeToolPath>
```

Comment:
```suggestion
      <_ALExeToolPath>$(TargetFrameworkSDKToolsDirectory)</_ALExeToolPath>
      <_ALExeToolPath Condition="'$(PlatformTarget)' == 'x64'">$(TargetFrameworkSDKToolsDirectory)$(PlatformTarget)\</_ALExeToolPath>
```

Reply:
Should `WinMdExpToolPath` have the same underscore prefix?

Reply:
Ideally, yes, but I don't think we should change it at this point.